libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog2.cs

```diff
@@ -0,0 +1,258 @@
+﻿//using System.Collections.Generic;
```

Comment:
I don't think this is right.

Reply:
You should remove the file...

---
In reply to: [412565861](https://github.com/microsoft/botbuilder-dotnet/pull/3775#discussion_r412565861) [](ancestors = 412565861)